src/tribler/core/conftest.py

```diff
@@ -43,15 +44,18 @@ def pytest_collection_finish(session):
 
 
 @pytest.hookimpl(hookwrapper=True)
-def pytest_runtest_protocol(item, log=True, nextitem=None):
+def pytest_runtest_protocol(item: Function, log=True, nextitem=None):
     """ Modify the pytest output to include the execution duration for all tests """
     # Perform the runtest protocol for a single test item.
-    start_time = time.time()
-    yield
-    duration = time.time() - start_time
-    total = time.time() - pytest_start_time
     if enable_extended_logging:
+        print(f'\nStarting "{item.name}"...', end='')
```

Comment:
**nitpick:** It may be better to add `flush=True` to print here to be sure it is printed even in case of a segmentation fault later in the test

Reply:
Added